tests/run_cross_cli_tests.py

```
@@ -0,0 +1,24 @@
+CMD = [
+    sys.executable,
+    "-m",
+    "unittest",
+    "-v",
+    "tests.test_cross_cli_status",
+]
+if os.environ.get("CROSS_CLI_LIVE") == "1":
+    CMD.append("tests.integration.test_cross_cli_live_tmux")
+
+print("$", " ".join(CMD))
+res = subprocess.run(CMD, cwd=str(REPO_ROOT), env=os.environ.copy())
```

Ruff (0.16.0)

[error] `subprocess` call: check for execution of untrusted input

(S603)